packages/contracts-bedrock/scripts/upgrades/holocene/templates/sc-ops-task/README.md

```diff
@@ -0,0 +1,33 @@
+# Holocene Hardfork Upgrade
+
+Status: DRAFT
+
+## Objective
+
+Upgrades the `SystemConfig` and Fault Proof contracts for the Holocene hardfork.
```

Comment:
Let's make sure this has placeholders to link to all the appropriate phases of the gov cycle, as done in other tasks such as: https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/eth/018-granite-upgrade#objective